### ISSUE TITLE
Favicon

### ISSUE THREAD

**Issue by morki** [mjl-/mox#186]
Webmail contains this html snippet in head:

```html
<link rel="icon" href="noneedlessfaviconrequestsplease:" />
```

I think this was meant to disable automatic favicon.ico request, but it is producing errors in chrome console:

![image](https://github.com/mjl-/mox/assets/3584190/e9e91a6b-c337-4c63-b7bf-beddc6063b69)

What is the reasoning behing this solution? Is there a problem with using normal favicon?
I would be glad if normal favicon is used even to easily locate webmail in pinned tabs.

**Comment by mjl-** [comment]
Mox doesn't have a logo or favicon. I don't know why, but browsers seem very aggressive in requesting favicons that don't exist. This was the only way I could easily find to prevent browsers from continuously making these useless requests all the time. Perhaps there is a better way? Another solution is making a proper favicon, but that's so low on my priority list I can't even see it. (:

**Comment by morki** [comment]
I think the current state is not good, because it also disallows users to use their favicon if they want to, for example with reverse proxy.

I created simple favicon with this tool https://favicon.io/favicon-generator/ and attaching here.

![image](https://github.com/mjl-/mox/assets/3584190/9d492ca9-2083-4472-81d8-84a311f1c65a)

You can of course create another, I don't have preferences (it can be then overriden in reverse proxy), just please use some favicon :)

[favicon_io.zip](https://github.com/user-attachments/files/16015392/favicon_io.zip)

**Comment by morki** [comment]
Now I see the same page has also emoji favicons prepared, like:

https://favicon.io/emoji-favicons/envelope
https://favicon.io/emoji-favicons/incoming-envelope/
https://favicon.io/emoji-favicons/closed-mailbox-with-lowered-flag/
https://favicon.io/emoji-favicons/closed-mailbox-with-raised-flag/
https://favicon.io/emoji-favicons/envelope-with-arrow/
https://favicon.io/emoji-favicons/e-mail/

This can be event better to not "hurry" for mox logo in general.

**Comment by mjl-** [comment]
Hi @morki, thanks for explaining the problem and giving the solution! It was indeed a bad situation that you couldn't configure a favicon at all. I removed the html snippets that prevented that. Browsers will be requesting /favicon.ico's again with this change, and mox is providing one that is builtin, as generated with favicon.io. I generated one with the ubuntu font (that is also used as font in the html pages), with a bright color to stand out. Mox can get a fancier logo at some later point.
